packages/babel-core/src/config/files/configuration.js

```diff
@@ -230,7 +231,21 @@ const readConfigJSON5 = makeStaticFileCache((filepath, content) => {
   try {
     options = json5.parse(content);
   } catch (err) {
-    err.message = `${filepath}: Error while parsing config - ${err.message}`;
+    // Check if the error contains source location informations
+    if (
+      typeof err.lineNumber === "number" ||
+      typeof err.columnNumber === "number"
```

Comment:
I don't think this is needed, or if we do want it, this should be `&&`ed, not `||`ed, since things could get weird if there's a column and no line.

Reply:
Oh sorry I wanted to ensure that both are present, I was a bit in a rush sorry.